aws-wafv2-loggingconfiguration/README.md

```diff
@@ -0,0 +1,29 @@
+# AWS::WAFv2::LoggingConfiguration
+This resource enables the creation of AWS WAFv2 Logging Configuration for WAFv2 WebACL via Cloudformation. In effect, you will now have the resource, AWS::WAFv2::LoggingConfiguration available to you to deploy using Cloudformation.
+
+This has been made possible using Cloudformation Registry. For more details about Cloudformation Registry - please look at: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/registry.html
+
+## Pre-Requisites
+
+1. Java Version 8 or higher
+2. Apache Maven - `brew install maven` - if you are on a Mac
+3. Cloudformation CLI and Cloudformation Java Plugin - https://docs.aws.amazon.com/cloudformation-cli/latest/userguide/what-is-cloudformation-cli.html
+The Cloudformation CLI tool is needed to submit this resource to Cloudformation Registry as a Private Type.
+
+## Submitting this resource as a Private Type
+
+The following instructions help you to extract the code in this repository and submit this resource as a private type to your account.
+
+1. Pull this code to your laptop using your favorite method. For me - `git clone git@github.com:advaj/aws-cloudformation-resource-providers-wafv2.git`
+2. Change to the Logging Configuration directory - `cd aws-cloudformation-resource-providers-wafv2/loggingconfiguration`
+3. Build this resource - `mvn clean package` - Make sure you are using Java 8 by running `java -version` on your laptop.
+4. Submit this resource to your account using - `cfn submit -vv --set-default`
+5. Go to Cloudformation -> Cloudformation Registry -> Private. You should see a new type - AWS::WAFv2::LoggingConfiguration
+
+Congratulations! You can now use AWS::WAFv2::LoggingConfiguration in your Cloudformation Templates. Example Templates are available in the repository
+
+## A few things to note -
+1. The Kinesis Firehose name must begin with aws-waf-logs
+2. Within RedactedFields, AllQueryArguments, Body and SingleQueryArgument are not supported
```

Comment:
Are we not supporting these fields in first release?

Reply:
The SDK does not seem to support AllQueryArguments, Body and SingleQueryArgument

Here is the error message that I get - 

`{'status': 'FAILED', 'errorCode': 'InvalidRequest', 'message': "Error reason: The parameter value isn't supported., field: FIELD_TO_MATCH, parameter: AllQueryArguments (Service: Wafv2, Status Code: 400, Request ID: 497d993a-688b-47d4-b799-12b15720062c, Extended Request ID: null)",`

`{'status': 'FAILED', 'errorCode': 'InvalidRequest', 'message': "Error reason: The parameter value isn't supported., field: FIELD_TO_MATCH, parameter: Body (Service: Wafv2, Status Code: 400, Request ID: 14ff808a-716b-4053-a867-318fb6516e8a, Extended Request ID: null)",`

`Failed to execute remote function: {Error reason: The parameter value isn't supported., field: FIELD_TO_MATCH, parameter: SingleQueryArgument (Service: Wafv2, Status Code: 400, Request ID: b298090c-8b47-4e11-830f-631fe8fcbb3a,`